troupon/payment/templates/payment/confirmation.html

```diff
@@ -2,22 +2,96 @@
 {% load static %}
 
 {% block main %}
-<div class="container">
+<div class="container vertical-center">
     <div class="row">
-        <div class="col-sm-12">
-            {% if messages %}
+        <div class="col-md-offset-2 col-md-8 ">
+            {%if messages%}
```

Comment:
**Code Quality: Nitpick**

Spaces between `{%' and the content. Same for closing `%}` tag.